Sming/Makefile

```diff
@@ -181,6 +181,10 @@ EXTRA_INCDIR    = include system/include Wiring Libraries SmingCore $(SDK_BASE)/
 THIRD_PARTY_DATA = third-party/rboot/Makefile
 EXTRA_INCDIR += third-party/rboot third-party/rboot/appcode 
 
+# => IRremote8266
+MODULES += Libraries/IR/src
```

Comment:
Things to improve in our build system: 
- The Arduino libraries c*, h files can be stored in the main root folder of the library or under `src`. We should change the build system to take care also for the second case.

Reply:
@horde3d With commit 58f0214cfbd33d6aca021103bdeed1e3d1a1544c that line is no longer needed. Please, rebase your fork and push back the changes. If you are not sure how then take a look at our https://github.com/SmingHub/Sming/blob/develop/CONTRIBUTING.md